editor/src/clj/editor/sentry.clj

```diff
@@ -139,33 +139,31 @@
   (-write [object out options] (json/-write (str object) out options)))
 
 (defn make-request-data
-  [{:keys [project-id key secret]} event]
-  {:request-method :post
-   :scheme         "https"
-   :server-name    "sentry.io"
-   :uri            (format "/api/%s/store/" project-id)
```

Comment:
I rewrote the HTTP client and simplified its API in the process. Now we simply supply a URL string instead of splitting it to scheme, server name etc.